sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
 # Release History
 
-## 1.23.0-beta.1 (Unreleased)
+## 1.23.0 (2022-04-05)
 
 ### Features Added
 
 - Added `AddClassifier` methods to `RequestContext`. These methods allow callers to change the response classification behavior for a given method invocation.
 - Added type `RequestOptions` to the `Azure` namespace and made `RequestContext` a subclass of `RequestOptions`.  This enables `RequestOptions` to be exposed in methods that take `CancellationToken` without causing confusion regarding which cancellation token will take effect.
 - Added the `SetUserAgentString` extension method to `HttpMessage` accepting a `UserAgentValue` and an optional application Id string. This allows assembly specific user agent header information to be set with proper formatting on a per-message basis.
+- Added an extension method to `BinaryData` called `ToObjectFromJson` which converts the json value represented by `BinaryData` to an object of a specific type.
```

Comment:
Do we want to add anything about `WaitUntil` or the other management plane chagnes?

Reply:
cc @m-nash

Reply:
I think adding anything about `WaitUntil` here would be out of context, the [samples around LRO](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md) should have these updates though for sure.

Reply:
https://apiview.dev/Assemblies/Review/77bdbcef49a3465fa96fac3fb745bf57/3547e2b6c9504989bbc669abdc5001ad?diffRevisionId=bf47f1ad7e134e81a1fefe7570186f0f&doc=False&diffOnly=True

Reply:
Ah yeah we can say we added more common public cloud references to `AzureLocation`